recipe/meta.yaml

```
@@ -1,5 +1,5 @@
 {% set name = "DiffPriv" %}
-{% set version = "1.0.0a0" %}
+{% set version = "1.0.0rc0" %}
```

Release candidate versions should start at `1`
```suggestion
{% set version = "1.0.0rc1" %}
```